targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L443xC/TOOLCHAIN_ARM/stm32l443xc.sct

```diff
@@ -102,7 +102,7 @@ LR_IROM1  MBED_APP_START  MBED_APP_SIZE  {    ; load region size_region
    .ANY (+RW +ZI)
   }
 
-  ARM_LIB_HEAP  AlignExpr(+0, 16)  EMPTY  (MBED_RAM_SIZE-RAM_FIXED_SIZE+MBED_RAM_START-AlignExpr(ImageLimit(RW_IRAM1), 16))  {
+  ARM_LIB_HEAP  AlignExpr(+0, 16)  EMPTY  (MBED_RAM_SIZE-RAM_FIXED_SIZE+MBED_IRAM2_START-AlignExpr(ImageLimit(RW_IRAM1), 16))  {
```

Comment:
This target has a very different memory layout - in fact, it has two RAMs at two distinct locations (see all lines above):
* RAM1 (address: MBED_RAM_START, size: MBED_RAM_SIZE):
  * some part of static memory (see comment at line 88)
  * heap (what we are calculating here)
  * stack (== RAM_FIXED_SIZE, defined at line 86)
* RAM2 (address: MBED_RAM2_START, size: MBED_RAM2_SIZE):
  * vector
  * crash report
  * remaining part of static memory (that didn't fit into RAM1), starting at MBED_**I**RAM2_START (they call that...)

So the old calculation was correct for this particular target.
```suggestion
  ARM_LIB_HEAP  AlignExpr(+0, 16)  EMPTY  (MBED_RAM_SIZE-RAM_FIXED_SIZE+MBED_RAM_START-AlignExpr(ImageLimit(RW_IRAM1), 16))  {
```